app/controllers/application_controller.rb

```diff
@@ -1,5 +1,24 @@
 class ApplicationController < ActionController::Base
   include Pundit
-
   protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  private
+
+  # Tell pundit how to find the current user
+  def pundit_user
+    current_admin
+  end
+
+  # Send a user to the admins index after sending invitations
+  def after_invite_path_for(inviter, invitee)
+    admins_path
+  end
+
+  # Customize which fields Devise allows for Admins
+  # See https://github.com/plataformatec/devise/tree/v3.5.2#strong-parameters
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:invite, keys: [:email, :role])
+  end
```

Comment:
These seem orthogonal to the API controller, and seem better encapsulated in the `AdminController`'s concerns. Why is it a better idea to have them here?

-- Govind and Srihari

Reply:
It's because `DeviseController` inherits from `ApplicationController`. It was either this or implement our own controllers for all of devise, which seemed excessive.